plugins/woocommerce/client/blocks/tests/e2e/tests/checkout/additional-fields.shopper.block_theme.spec.ts

```diff
@@ -566,6 +570,36 @@ test.describe( 'Shopper → Additional Checkout Fields', () => {
 				}
 			);
 
+			await checkoutPageObject.page
+				.getByLabel( 'Alternative Email' )
+				.fill( 'test@' );
+
+			// Blur the field to trigger the validation.
+			await checkoutPageObject.page.evaluate(
+				'document.activeElement.blur()'
```

Comment:
why not

```js
await checkoutPageObject.page
	.getByLabel( 'Alternative Email' )
	.blur();
```

Reply:
Honestly, didn't know it had that method available.

Reply:
https://github.com/woocommerce/woocommerce/pull/57361/commits/8c19bfef13bc0c0955a47bf0ab3acb0c91bdac75